### ISSUE TITLE
Plot sjp.int on Logit Scale

### ISSUE THREAD

**Issue by nkcaselli** [sjPlot/devel#157]
I submitted a plot of an interaction between two continuous variables in a logistic regression in a manuscript, and the reviewers asked that I plot the interaction on a logit scale. Is there a way to do this in this package?

The syntax I used for the model was:
baseline<-glmer(Acquired ~ Age.Z.*SignFrequency.Z.+Age.Z.*Iconicity.Z.+Age.Z.* NeighborhoodDensity.Z.+(1+SignFrequency.Z.+Iconicity.Z.+NeighborhoodDensity.Z.|SubjectNumber)+(1| EntryID),data= nominor, family = binomial, nAGQ=0, control = glmerControl(optimizer = "nloptwrap"))

The syntax I used for the plot was: 
sjp.int(baseline, type = "eff", int.term = "Age.Z.*Iconicity.Z.",mdrt.values = "quart",show.ci=TRUE)

**Comment by sjPlot** [comment]
Yes, the `effect()` function, which is used by `sjp.int()`, has a `transformation` argument. You can pass down these arguments to `effect()`. If `tranformation = NULL`, no inverse-link function is used to plot predicted values, but to use the original scale (in your case: logits).

```
library(sjmisc)
data(efc)
# create binary response
y <- ifelse(efc$neg_c_7 < median(stats::na.omit(efc$neg_c_7)), 0, 1)
# create data frame for fitted model
mydf <- data.frame(y = as.factor(y),
                   sex = as.factor(efc$c161sex),
                   barthel = as.numeric(efc$barthtot))
# fit model
fit <- glm(y ~ sex * barthel, data = mydf, family = binomial(link = "logit"))
# plot interaction, increase p-level sensivity
sjp.int(fit, type = "eff", transformation = NULL)
# same as
plot(effects::effect("sex*barthel", fit, transformation = NULL))
```

You just have to change the y-scale:

```
library(ggplot2)
p <- sjp.int(fit, type = "eff", transformation = NULL)
p$plot.list[[1]] + scale_y_continuous(name = "logits")
```

Does this work for you?

**Comment by nkcaselli** [comment]
This is fantastic. Yes, it seems to work. I really appreciate your help.

**Comment by sjPlot** [comment]
I have changed all effect-plots in the sjp.function, which support glm(er), including `sjp.int()` - when you now use the `transformation`-argument, the scale and scale-title are automatically changed accordingly. Will commit to GitHub later this day.
